routes/quoteCalc.js

```diff
@@ -1,9 +1,28 @@
 const express = require("express");
 const router = express.Router();
 
-//Below is for get, change get to post for post request
-router.get("api url", (req, res) => {
-  //Api code
+function calculatePremium(car_value, risk_rating) {
```

Comment:
Why is this function duplicated here?

Reply:
Thanks for spotting - removed from quoteCalc.js in /api.